src/lib/sns.js

```diff
@@ -0,0 +1,45 @@
+import { sns } from './aws-clients.js'
+import logger from './logger.js'
+import { isCollection, isItem } from './stac-utils.js'
+
+const attrsFromPayload = function (payload) {
+  let type = 'unknown'
+  let collection = ''
+  if (isCollection(payload.record)) {
+    type = 'Collection'
+    collection = payload.record.id || ''
+  } else if (isItem(payload.record)) {
+    type = 'Feature'
```

Comment:
I think it's going to be more understandable here if we set the type to Collection or Item, to map it to the STAC object type rather than the literal value of the `type` field of the item ingested.

Reply:
@jkeifer Phil made a good suggestion regarding the message attributes that we include when publishing ingest notifications to SNS. 

>I think it's going to be more understandable here if we set the type to Collection or Item, to map it to the STAC object type rather than the literal value of the type field of the item ingested.

Using `Item` instead of `Feature` seems more correct, but it would introduce a change to the attribute schema that you proposed in #131 and had mentioned was in use in a production deployment. I am planning to got with `Item` as Phil suggested and wanted to make sure you were aware of it.